qp_woltka/woltka.py

```diff
@@ -33,7 +33,7 @@
 MEMORY = '100g'
 LARGE_MEMORY = '180g'
 MERGE_MEMORY = '80g'
-SYNDNA_MEMORY = '190g'
+SYNDNA_MEMORY = '5g'
```

Comment:
From 190gb to 5gb here as well.

Reply:
It's the same value, here is where is defined and the other is the test.